packages/jest-snapshot/src/snapshot_resolver.ts

```diff
@@ -65,8 +73,11 @@ function createDefaultSnapshotResolver(): SnapshotResolver {
 
 function createCustomSnapshotResolver(
   snapshotResolverPath: Config.Path,
+  localRequire: (moduleName: string) => SnapshotResolver,
 ): SnapshotResolver {
-  const custom: SnapshotResolver = require(snapshotResolverPath);
+  const custom: SnapshotResolver = interopRequireDefault(
```

Comment:
shouldn't need `interopRequireDefault` now, right? just pass `true` as second arg?

Reply:
I wondered the same thing, but it's an argument: https://github.com/facebook/jest/pull/8829/files/e796781ba2f3d59d261d429c52c7ae2e7e0c807c#diff-79e46c9e92027daadc4fc85b7b71ae2dR30

So we can't assume that the second arg will trigger that behaviour 😞